dist/index.html

```diff
@@ -232,7 +232,27 @@ <h2>Fitness</h2>
           Blog Posts
         </div>
         <div class="footer-section__feed">
-          instagram or twitter news feed
+          <div class="row_header">
+            <p>INSTAGRAM</p>
```

Comment:
I recommend you to use <h3> instead <p>
you could find more about here:
https://www.w3schools.com/tags/tag_p.asp
https://www.w3schools.com/tags/tag_hn.asp

Reply:
Thanks for your suggestion i edited it and commited again.